docs/subquery_network/kepler/indexers/become-an-indexer.md

```diff
@@ -100,6 +99,10 @@ To index a project, please follow the instructions [here](../indexers/index-proj
 If you are wanting to index a Dictionary, then you may be able to restore your project from our dictionary snapshots to save a lot of time. Instructions are [here](../indexers/dictionary-restore.md)
 :::
 
+## 5. Create a Plan from a Plan Template
+
+For the intial stages of Kepler, the SubQuery Council will set some default plan templates to make plan management and creation easier, and speed up the test and analysis of various parameters of the network. As a result, plan creation is very easy and just requires to you enter a price. [See the guide here](./plans.md#creating-a-plan).
```

Comment:
I'm thinking a separated page for indexer sponsor program.
then we can explain more about it.
1. the whole period
2. period of each rounds
3. criteria
  1. project status ready,
  2. avaliability
  3. price <= max price
5. all factors that affect the score
6. where and when we will announce the picking for each round

Reply:
Added